static/usr/lib/systemd/system/remount-core-fs.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Reset propagation of initial mount points
+DefaultDependencies=no
+Before=local-fs-pre.target
+Before=local-fs.target
+Before=shutdown.target
+Wants=local-fs-pre.target
+Conflicts=shutdown.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/lib/core/remount-core-fs
+
+[Install]
```

Comment:
Install section is not needed and it looks like in general we don't add it unless it is the case.

Reply:
I know. I am not sure how I feel about the way we do it. We put everything in `/usr/lib/systemd` because `/etc/systemd/system` is writable. But we can always symlink to `/dev/null` in `/etc` to mask services installed in `/usr`. So the reason we use `/usr/lib/systemd` does not completely work.

I have added the `Install` section as indication of where it is expected to be installed. And in that way, we could one day move to another approach than what we are doing.

I can remove it.

Reply:
I actually prefer it to stay as I agree it is better to have it as a hint on where the service is expected. But then we should enforce it in next changes as good practice.

Reply:
I have created #86 so we only use install sections and generate the symlinks.